### PR TITLE
Feat: Github actions for CI

### PR DESCRIPTION
**Pull request by jellypuno**
Signed-off-by: Jessielaine Punongbayan <jessielaine.punongbayan@broadcom.com>

Steps included in the CI:
1. Check-out
2. Install
3. Pre-Publish
4. Audit
5. Run Tests

Tested OR: Windows, Ubuntu and MacOS (Latest)
Tested NodeJS: 10 and 12
Tested branches: Master and Development

### CODE REVIEW

**Comment by jellypuno** [comment]
@Alexandru-Dumitru @zdmullen @crawr Kindly review this PR. Please let me know if the CI process is correct.

**Comment by jellypuno** [comment]
note: CI fails on npm audit

File: .github/workflows/Explorer-for-Endevor-CI.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install
+
+      - run: npm run vscode:prepublish
```
Comment (Alexandru-Dumitru) [inline .github/workflows/Explorer-for-Endevor-CI.yml:36]: I think this should be `npm run compile` for the time being. I know it's executing the same command, but we can be verbose about the action

**Review by Alexandru-Dumitru — CHANGES_REQUESTED** [review]
Ran the tests locally. All is good for me. I requested one small change in a comment above.

About the audit issues, not sure how to handle it. Ideally we should resolve them, but we can just do it for 1 at this time. I guess we can accept it.

Thank you.

**Review by Alexandru-Dumitru — APPROVED** [review]
LGTM. Thanks Jelly for implementing this and addressing my comments.
As agreed, we skip low level audit findings.

**Pull request merged**